jena-base/src/main/java/org/apache/jena/atlas/lib/CollectionUtils.java

```diff
@@ -31,6 +32,10 @@
 
     /** Return an element from a collection. */ 
     public static <T> T oneElt(Collection<T> collection) {
+        if ( collection.isEmpty() )
```

Comment:
Should this method have a `null` check for `collection`?

Reply:
Done (the calls were safe anyway).